src/validators/common.ts

```diff
@@ -214,6 +214,13 @@ export type IdentifierSection = Record<string, IdentifierT & {
 	type: number;
 }>;
 
+export type SubmissionSection = {
+	note?: string;
+	annotation?: {
+		content: string;
+	};
```

Comment:
Something is odd here: The validators expect an optional `submissionSection.annotation.content` property, but the website has an `annotationSection` as well... Looks like the validators are out of date?
If that is the case, both the validators and this type definition should be adapted.

Reply:
Indeed, in the website I only see annotationSection.content and the submissionSection does not contain any annotation data.

Reply:
It seems `submissionSection.annotation` was a relict from #448, which subsequently was replaced by #450. Only the old validation functions stayed behind.
I've now deleted them in c0570ae after having added proper validation functions (and accompanying tests) for the `annotationSection` in 45ce6c9.